src/App.js

```diff
@@ -2,29 +2,28 @@ import React from 'react';
 
 import './App.css';
 
-import posts from './api/posts';
-import comments from './api/comments';
-import users from './api/users';
+import ApiPosts from './api/ApiPosts';
+import ApiComments from './api/ApiComments';
```

Comment:
these two files are not components, name should start with lowercase letter as usual